genai-perf/docs/tutorial.md

```diff
@@ -28,192 +28,121 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # Profile Large Language Models with GenAI-Perf
 
-- [Profile GPT2 running on Triton + TensorRT-LLM](#tensorrt-llm)
-- [Profile GPT2 running on Triton + vLLM](#triton-vllm)
-- [Profile GPT2 running on OpenAI Chat Completions API-Compatible Server](#openai-chat)
-- [Profile GPT2 running on OpenAI Completions API-Compatible Server](#openai-completions)
-
----
-
-## Profile GPT2 running on Triton + TensorRT-LLM <a id="tensorrt-llm"></a>
-
-### Run GPT2 on Triton Inference Server using TensorRT-LLM
-
-<details>
-<summary>See instructions</summary>
-
-Run Triton Inference Server with TensorRT-LLM backend container:
+This tutorial will demonstrate how you can use GenAI-Perf to measure the performance of
+various inference endpoints such as
+[KServe inference protocol](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2)
+and [OpenAI API](https://platform.openai.com/docs/api-reference/introduction)
+that are widely used across the industry.
 
-```bash
-export RELEASE="yy.mm" # e.g. export RELEASE="24.06"
+### Table of Contents
 
-docker run -it --net=host --gpus=all --shm-size=2g --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/tritonserver:${RELEASE}-trtllm-python-py3
+- [Profile GPT2 running on Triton + TensorRT-LLM Backend](#tensorrt-llm)
+- [Profile GPT2 running on Triton + vLLM Backend](#triton-vllm)
+- [Profile GPT2 running on OpenAI Chat Completions API-Compatible Server](#openai-chat)
+- [Profile GPT2 running on OpenAI Completions API-Compatible Server](#openai-completions)
 
-# Install Triton CLI (~5 min):
-pip install "git+https://github.com/triton-inference-server/triton_cli@0.0.8"
+</br>
 
-# Download model:
-triton import -m gpt2 --backend tensorrtllm
+## Profile GPT-2 running on Triton + TensorRT-LLM <a id="tensorrt-llm"></a>
 
-# Run server:
-triton start
-```
-
-</details>
+You can follow the [quickstart guide](https://github.com/triton-inference-server/triton_cli?tab=readme-ov-file#serving-a-trt-llm-model)
+on the Triton CLI github repo to serve GPT-2 model on the Triton server with TensorRT-LLM backend.
```

Comment:
```suggestion
in the Triton CLI repository to serve GPT-2 model on the Triton server with TensorRT-LLM backend.
```

Reply:
thanks for catching this. Updated the sentence